source/includes/steps-create-collection.yaml

```diff
@@ -19,6 +19,17 @@ content: |
    </reference/collation/#collation-document>` on the collection,
    select the :guilabel:`Use Custom Collation` checkbox and select the
    desired collation settings.
+
+   If your deployment is connected using :guilabel:`In-Use Encryption`, you can 
+   use :v6.0:`Queryable Encryption </core/queryable-encryption/>` on the newly 
+   created collection. Select the :guilabel:`Queryable Encryption` checkbox
+   and indicate the following:
+
+   - :v6.0:`Encrypted Fields </core/queryable-encryption/fundamentals/encrypt-and-query/>`.
+
+   - (Optional) :v6.0:`KMS Provider </core/queryable-encryption/fundamentals/kms-providers/>`.
+
+   - (Optional) :v6.0:`Key Encryption Key </core/queryable-encryption/fundamentals/keys-key-vaults/>`. 
```

Comment:
[future consideration]

I think what we have here is pretty good, but we might want to consider separating these specific collection types into individual task-pages, each with a more complete example, and make them sub-pages of the "Collection" page.

I'm kind of imagining something like:

- Databases
- Collections
  - Create a Collection
  - Create a Capped Collection
  - Create a Collection with Custom Collation
  - Create a Collection with Encrypted Fields

Something like this might make the information a bit more discoverable, and users who aren't interested in these topics won't be confused by extra info that doesn't apply to them.

If you think something like this is a good idea, maybe create a follow-up ticket where we can revisit some of the structure of these pages. I'm happy to hop on a call to discuss this in a bit more detail. To clarify, this is a non-blocking suggestion for this PR and would definitely be suited to a follow-up effort.

Reply:
I agree and can go ahead and make a ticket to follow-up on this.